api/v2/serializers/details/__init__.py

```diff
@@ -1,4 +1,6 @@
 from .allocation import AllocationSerializer
+from .allocation_request import AllocationRequestSerializer,\
```

Comment:
I don't think we have decided on the convention. I've been sticking with \ because majority of the could still uses the \ style convention.